README.md

```diff
@@ -56,14 +57,32 @@ $ cargo test-bpf  # <-- runs BPF program tests
 Integration testing may be performed via the per-project .js bindings.  See the
 [token program's js project](token/js) for an example.
 
+### Common Issues
+Solutions to a few issues you might run into are mentioned here.
+
+1. `Failed to open: ../../deploy/spl_associated_token_account.so`
+    
+    Update your Rust and Cargo to the latest versions and re-run the `cargo build` and `cargo build-bpf` steps.
+    Make sure you have Solana version **v1.10.19 or later**.
+
+2. [Error while loading shared libraries. (libssl.so.1.1)](https://github.com/project-serum/anchor/issues/1831)
+
+    A working solution was mentioned [here](https://github.com/project-serum/anchor/issues/1831#issuecomment-1109124934).
```

Comment:
Let's copy the full solution here in case something weird ever happens to the link:

```
wget http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1l-1ubuntu1.2_amd64.deb
sudo dpkg -i libssl1.1_1.1.1l-1ubuntu1.2_amd64.deb
```